### PR TITLE
PS-8736 Update the 8.0.32 release notes

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/release-notes/8.0.32-24.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]

File: docs/release-notes/8.0.32-24.md [hunk 1]
```diff
@@ -32,15 +32,17 @@ increased visibility.
...
+    The Percona Server version of the `mysqldump` utility, in some modes, can be used with MySQL Server. This utility provides a temporary workaround for the "additional RELOAD privilege" limitation introduced by Oracle MySQL Server 8.0.32. 
 
-For more information, see the Percona Performance Blog [A Workaround for the "RELOAD/FLUSH_TABLES privilege required" Problem When Using Oracle mysqldump 8.0.32](https://www.percona.com/blog/workaround-for-the-reload-flush_tables-privilege-required-problem-when-using-oracle-mysqldump-8-0-32/).
+    For more information, see the Percona Performance Blog [A Workaround for the "RELOAD/FLUSH_TABLES privilege required" Problem When Using Oracle mysqldump 8.0.32](https://www.percona.com/blog/workaround-for-the-reload-flush_tables-privilege-required-problem-when-using-oracle-mysqldump-8-0-32/).
+
+* Percona Server for MySQL has changed the default for [supported DDL column operations](https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html) from `ALGORITHM=INPLACE` to `ALGORITHM=INSTANT`.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.32-24.md]: I would put the whole context here...

As all most important bugs related to ALGORITHM=INSTANT were fixed by Oracle in MySQL Server 8.0.32, Percona Server for MySQL 8.0.32 returned original(upstream's) behaviour. For supported DDL column operations the default ALTER TABLE algorithm is now INSTANT again. It was previously deliberately changed  to INPLACE in Percona Server for MySQL 8.0.29 for safety reasons (see this blog post for more details https://www.percona.com/blog/percona-xtrabackup-8-0-29-and-instant-add-drop-columns/).

File: docs/release-notes/8.0.32-24.md [hunk 2]
```diff
@@ -32,15 +32,17 @@ increased visibility.
 
 ## Release highlights
 
-Percona decided to revert the following MySQL bug fix:
+* Percona decided to revert the following MySQL bug fix:
```
Comment (patrickbirch) [inline docs/release-notes/8.0.32-24.md]: Percona reverts

File: docs/release-notes/8.0.32-24.md (hunk 1 above)
Comment (patrickbirch) [inline docs/release-notes/8.0.32-24.md]: Percona Server for MySQL changes

File: docs/release-notes/8.0.32-24.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.32-24.md]: fixed

File: docs/release-notes/8.0.32-24.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.32-24.md]: rewritten

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.32-24.md]: Updated

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**